pages/code-review.md

```diff
@@ -24,10 +24,34 @@ goals when reviewing code are to:
 - cultivate the ability to critique one’s code more strongly;
 - encourage open communication between the entire team.
 
+## Discuss code review with your team
+
+Every individual and team has a different set of expectations around code
+review. Discussing these expectations explicitly is the best way to make sure
+that everyone is on the same page.
+
+Bonus points for documenting and/or automating the outcomes of your discussion.
+
+Some guiding questions for your team's code review discussion:
+
+- What does the current code review process, at a high level, look like? Start
+  with the feature request and discuss each step until a feature is considered
+  "done"
+- How are code reviewers assigned?
+- How many people are expected to review each pull request?
+- Does your team make use of any tools to aid in code reviews? (eg: CI, Static
+  Analysis, linters)
+- Are there any automated tools that you are not using that you would like to try?
+- What do you look for when conducting a code review?
+- How do you know that a pull request is ready to be merged?
+- Who merges code, the author or the reviewer?
+- What about the current process do you like?
+- What about the current process do you think can be improved?
+
```

Comment:
A few more potential questions, mostly refinements of your existing bullets
* Should code be pulled down and tested/what QA strategy is used?
* Are pull requests assigned? If so what process is used?
* If a patch is worked on in a pairing session, how does it get merged?
* What do we do about automated tool failures? For example, if code climate doesn't like something, can we merge anyway or are we strict?
* How do changes and refinements to this process get made? Retro? PR to a contributing file?

Not strictly code review, but worth hammering out at the same time
* Forks or branches?
* What's the git workflow? E.g. is master always deployable? Rebase or merge?